test/readme.md

```diff
@@ -115,38 +114,65 @@ Then run the tests:
 npm run check:test
 ```
 
-> [!NOTE]
-> The command above will run a subset of the tests that work with the standalone server topology since the tests are being run against a standalone server.
+> **Note:** the command above will run a subset of the tests that work with the standalone server topology since the tests are being run against a standalone server.
 
 The output will show how many tests passed, failed, and are pending. Tests that we have indicated should be skipped using `.skip()` will appear as pending in the test results. See [Mocha's documentation][mocha-skip] for more information.
 
 In the following subsections, we'll dig into the details of running the tests.
 
+### Testing Against Different Versions
+
+Sometimes you'll need to run tests against a specific version, such as "7.0", and to do that you can use the `VERSION` env var.
+
+```sh
+VERSION=7.0 TOPOLOGY='replica_set' bash .evergreen/run-orchestration.sh
+```
+
+Older versions of MongoDB may not have Mac binaries, so you may also need to specify a different architecture:
+
+```sh
+VERSION=4.2 ARCH=x86_64 TOPOLOGY='replica_set' bash .evergreen/run-orchestration.sh
```

Comment:
The architecture here is what's causing the need for Rosetta to be installed. ARCH on Apple silicon should be arm64 to get the correct binaries.